backends/p4tools/testgen/targets/bmv2/test/BMV2PTFXfail.cmake

```diff
@@ -15,3 +15,9 @@ p4tools_add_xfail_reason(
   # Assert/Assume error: assert/assume(false).
   bmv2_assert.p4
 )
+
+p4tools_add_xfail_reason(
+  "testgen-p4c-bmv2-ptf"
+  "Key does not have a name annotation."
+  bmv2_restrictions.p4
```

Comment:
I would fix up this program instead of adding it to xfails. Why does the key here not have an annotation?

Reply:
I don't quite understand, is the name annotation required for table keys?

Reply:
If it is not present it generally is always added, I am curious why that does not happen here.